samples/bluetooth/hci_uart/nrf52840_pca10090.overlay

```diff
@@ -0,0 +1,15 @@
+/ {
+	chosen {
+		zephyr,bt-uart=&uart1;
+	};
+};
+
+&uart1 {
```

Comment:
How do we solve the problem with uart1 being enabled in the nrf9160_pc10090 by default?

Reply:
Right, my mistake.